f/medical/f_CLSEH.sqf

```diff
@@ -0,0 +1,39 @@
+// FA3 - Combat Life Saver Event Handler
+// This component adds an event handler for use with the Combat Life Saver assignGear class. Units with the f3_cls trait can provide full heals using FAKs.
+// Enable this component in init.sqf
+// For credits and more information see https://github.com/folkarps/F3/wiki
+
+// Make sure the player can be broadcast properly
+_unit = player;
```

Comment:
Might want to have a sleep or an isNull player waituntil here. Is null players in the pregame still a thing?

Reply:
Added initialisation delay as used in other F3 components